back/engines/commercial/multi_tenancy/db/seeds/tenants.rb

```diff
@@ -52,6 +52,14 @@ def create_localhost_tenant
               client_id: ENV.fetch('DEFAULT_GOOGLE_LOGIN_CLIENT_ID'),
               client_secret: ENV.fetch('DEFAULT_GOOGLE_LOGIN_CLIENT_SECRET')
             },
+            azure_ad_login: {
+              allowed: true,
+              enabled: true,
+              tenant: ENV.fetch('DEFAULT_AZURE_AD_LOGIN_TENANT_ID'),
+              client_id: ENV.fetch('DEFAULT_AZURE_AD_LOGIN_CLIENT_ID'),
+              logo_url: 'https://res.cloudinary.com/citizenlabco/image/upload/v1599721853/Product/microsoft-azure-logo.png',
```

Comment:
Sorry, I forgot to mention. Not a big deal probably, but I think we usually try to use s3 for files in the repo.

Reply:
Good point, I forgot we made this change. I'll use the `cl2-seed-and-template-assets` bucket instead.